hive/src/test/java/org/apache/iceberg/hive/HiveTableTest.java

```diff
@@ -295,4 +302,29 @@ public void testListTables() {
     Assert.assertEquals(1, expectedIdents.size());
     Assert.assertTrue(catalog.tableExists(TABLE_IDENTIFIER));
   }
+
+  @Test
+  public void testNonDefaultDatabaseLocation() throws IOException, TException {
+    // Create a new location and a non-default database / namespace for it
+    File nonDefaultLocation = createTempDirectory(NON_DEFAULT_DATABASE,
+        asFileAttribute(fromString("rwxrwxrwx"))).toFile();
+    Database database = new Database();
+
+    database.setName(NON_DEFAULT_DATABASE);
+    database.setLocationUri(nonDefaultLocation.getPath());
+    metastoreClient.createDatabase(database);
```

Comment:
The catalog supports creating a namespace, so I think this test would be better if it used the catalog to create the namespace with a location ("location" property) instead of going through the metastore client directly. Could you update it?

Reply:
Sure! Updated.
Thanks for the review!